src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -1582,6 +1591,13 @@ public static void createAllDirectories()
                     throw new ConfigurationException("cdc_raw_directory must be specified", false);
                 FileUtils.createDirectory(conf.cdc_raw_directory);
             }
+
+            boolean created = maybeCreateHeapDumpPath();
+            if (!created && conf.dump_heap_on_uncaught_exception)
+            {
+                logger.error(String.format("conf.dump_heap_on_uncaught_exception is enabled but unable to create heap dump path %s. Disabling.", conf.heap_dump_path != null ? conf.heap_dump_path : "null"));
```

Comment:
nit: In the user-facing error message, I guess I would say `dump_heap_on_uncaught_exception` rather than `conf.dump_heap_on_uncaught_exception`

Reply:
...or I guess you could say "cassandra.yaml:dump_heap_on_uncaught_exception" like you do in a couple other places

Reply:
👍 agree; changed.